docs/faq.txt

```diff
@@ -70,6 +70,8 @@ wrap the script in ``<pre>`` tags to properly format its output:
 
 .. code-block:: php
 
+   <?php
+
    <pre><?php require(...); ?></pre>
```

Comment:
This change can be removed, as the current docs correctly pick up the opening tags within the `<pre>` tags (screenshot is from production docs):
![Screenshot from production docs](https://user-images.githubusercontent.com/383198/212853141-68664b26-7be2-413a-91fc-bd01b82decb6.png)

```suggestion
   <pre><?php require(...); ?></pre>
```